examples/step-63/step-63.cc

```diff
@@ -390,14 +440,18 @@ namespace Step63
       values[i] = BoundaryValues<dim>::value(points[i], component);
   }
 
+
+
+  // @sect3{Streamline Diffusion}
+
+  // Streamline diffusion stabilization term. Value defined in
+  // https://link.springer.com/chapter/10.1007/978-3-540-34288-5_27.
```

Comment:
Also, is this acceptable? Or should I include author/title?

Reply:
yes, include authors and title here

Reply:
Maybe in the same style as above (makes it easy to click and arrive directly at the paper).